Egy Le Rendszere/Ajánlatok

```diff
@@ -0,0 +1,25 @@
+Nézvén a magaslatból
+Látod hogy a hajók messze
+Látod sasszemmel
+A hullámok folyton csapkodnak
+Ha te vagy a jelfény
+Tiéd a munkásosztály
+De hogyha válasz kéne
+Jobb lesz ha adsz egy darab segget!
+Adj egy darab segget
+
+Vigyázz, delej utáni tippek
+Hajót partravető
+A narancs fény mely követ
+Istenné teszi majd magát
+Ha célzod kérdésed
+Felemészt majd téged a köd
+De hogyha válasz kéne
+Jobb lesz ha felkészülsz a tűzre!
+Felkészül a tűzre
+
+A hajók napról napra többek, uram
```

Comment:
Itt is van 1 szótagnyi hely
Ajánlat:
"A hajók egyre többen vannak, uram" (nem old meg semmit, csak talán magyarosabb)

Reply:
az is tényező hogy a sir helyett az uram két szótag